TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/dialog/PreviewAppearanceDialog.java

```diff
@@ -0,0 +1,57 @@
+package com.ybsystem.tweetmate.fragments.dialog;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
+import android.view.WindowManager;
+
+import androidx.fragment.app.DialogFragment;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentTransaction;
+
+import com.ybsystem.tweetmate.R;
+import com.ybsystem.tweetmate.fragments.timeline.ProfileTimeline;
+import com.ybsystem.tweetmate.models.entities.Column;
+
+import static com.ybsystem.tweetmate.models.enums.ColumnType.USER_TWEET;
+
+public class PreviewAppearanceDialog extends DialogFragment {
+
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+
+        // Show full screen
+        Window window = getDialog().getWindow();
+        WindowManager.LayoutParams params = window.getAttributes();
+        params.width = ViewGroup.LayoutParams.MATCH_PARENT;
+        params.height = ViewGroup.LayoutParams.MATCH_PARENT;
+        window.setAttributes(params);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate
+        View view = inflater.inflate(R.layout.dialog_preview_appearance, null);
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ybs32/TweetMate-for-Android/blob/0a5013e538213bacb2774f24bfa4ed6959fa720c/TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/dialog/PreviewAppearanceDialog.java#L37"></span>Avoid passing <code>null</code> as the view root (needed to resolve layout parameters on the inflated layout's root element)</td>
    </tr>
  </tbody>
</table>